pkg/controller/pcidevice/pcidevice_controller.go

```diff
@@ -61,17 +63,26 @@ func Register(
 
 func (h Handler) reconcilePCIDevices(nodename string) error {
 	// List all PCI Devices on host
+	pci, err := ghw.PCI()
+	if err != nil {
+		return err
+	}
+	// Build up the IOMMU group map
+	iommuGroupPaths, err := iommu.GroupPaths()
+	if err != nil {
+		return err
+	}
+	iommuGroupMap := iommu.GroupMapForPCIDevices(iommuGroupPaths)
 
 	commonLabels := map[string]string{"nodename": nodename} // label
 	var setOfRealPCIAddrs map[string]bool = make(map[string]bool)
-	for _, dev := range h.pci.Devices {
+	for _, dev := range pci.Devices {
 		if !containsString(h.skipAddresses, dev.Address) {
 			setOfRealPCIAddrs[dev.Address] = true
 			name := v1beta1.PCIDeviceNameForHostname(dev, nodename)
 			// Check if device is stored
 			var err error
```

Comment:
```suggestion
``````suggestion
			var err error
```
not needed as we are already initialising err in the next line